platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/ivyresolve/ErrorHandlingModuleComponentRepository.java

```diff
@@ -61,10 +61,12 @@ public class ErrorHandlingModuleComponentRepository implements ModuleComponentRe
     private final ErrorHandlingModuleComponentRepositoryAccess local;
     private final ErrorHandlingModuleComponentRepositoryAccess remote;
 
-    public ErrorHandlingModuleComponentRepository(ModuleComponentRepository<ModuleComponentGraphResolveState> delegate, RepositoryDisabler remoteRepositoryBlacklister) {
+    private final static String DISABLED_REPOSITORY_ERROR_MSG_TEMPLATE = "Repository %s is disabled " + BuildExceptionReporter.SKIPPABLE_ERROR_MESSAGE_SUFFIX;
```

Comment:
💭 I don't really like the distance between the format string and the actual call, because it's easy to change one without the other and not notice it. Additionally, it makes it harder to read what the message is supposed to be at the format call site. Due to it being used multiple times, perhaps this should be a `static` method that takes the name it needs?

Reply:
Good idea, I'll make this a method.